lib/textris/base.rb

```diff
@@ -53,9 +42,15 @@ def call_action
     end
 
     def render_content
-      set_instance_variables_for_rendering
-
-      render(:template => template_name, :formats => ['text'], :locale => @locale)
+      renderer = ::ApplicationController.renderer.new
```

Comment:
👋 I just tested this PR on a rails api I am upgrading to rails 6 and `renderer.render` keeps returning an empty string because views are not loaded in rails apis. The controllers in rails apis inherit from `ApplicationController::API` instead of `ApplicationController::Base`. I was wondering if we could switch to using 
```ruby
ActionController::Base.new.render_to_string(
  template_name,
  layout: false,
  locale: @locale,
  assigns: set_instance_variables_for_rendering
)
```
which works on both Rails apis and vanilla rails apps

Reply:
Thanks for the input @no-itsbackpack. 

True, didn't think about that. I used `ApplicationController` as Helpers are automatically available this way. 

Not sure how to achieve this with `ActionController::Base` though. Going back to using...

```ruby
module Textris
  class Base
    class RenderingController < ActionController::Base
      helper :all
    end

  # ...

  def render_content
    renderer = RenderingController.renderer.new

  # ...
```

... does not seem to load helpers defined in the application.

Reply:
Yea I totally didnt consider loading controller helpers, back to the drawing board 🤔